### ISSUE TITLE
Javascript Error

### ISSUE THREAD

**Issue by denzelchia** [presscustomizr/hueman-addons#1]
Hi,

Just installed the plugin and notice there is a javascript error.
Please see screenshot. 
http://i.imgur.com/F27dONv.png

This is screenshot of my setting.
http://i.imgur.com/RIIbTmb.png

I can only see the a plus sign and twitter share icon which does not work, no other icons.
Probably broken due to javascript error.
http://i.imgur.com/bmckDKc.jpg

The following is my system information

Thank you!
Denzel

```
# SITE_URL:                 http://localhost:8888/hueman-lite
# HOME_URL:                 http://localhost:8888/hueman-lite
# IS MULTISITE :            No

# THEME | VERSION :         Hueman | v3.0.10# WP VERSION :              4.5
# PERMALINK STRUCTURE :     /%year%/%monthnum%/%day%/%postname%/

# ACTIVE PLUGINS :
Hueman Addons: 1.0.0

PHP Version:              5.6.10
MySQL Version:            5.5.42
Web Server Info:          Apache/2.2.29 (Unix) mod_wsgi/3.5 Python/2.7.10 PHP/5.6.10 mod_ssl/2.2.29 OpenSSL/0.9.8zh DAV/2 mod_fastcgi/2.4.6 mod_perl/2.0.9 Perl/v5.22.0

WordPress Memory Limit:   40MB
PHP Safe Mode:            No
PHP Memory Limit:         256M
PHP Upload Max Size:      32M
PHP Post Max Size:        32M
PHP Upload Max Filesize:  32M
PHP Time Limit:           30
PHP Max Input Vars:       1000
PHP Arg Separator:        &
PHP Allow URL File Open:  Yes
WP_DEBUG:                 Enabled

Show On Front:            posts
Page On Front:             (#0)
Page For Posts:            (#0)
```

**Comment by Nikeo** [comment]
ok thanks for reporting this.
I'm on it

**Comment by denzelchia** [comment]
Hi,

I think this issue is cause by the folder name.

I downloaded hueman-addons-master.zip from github download zip button.
After installing the plugin, the folder name is hueman-addons-master which I think causes the javascript error. I renamed the folder to hueman-addons and re-activated the plugin.
Now it works. No more javascript error.

Sorry for causing inconveniences! I did not notice the download link in the readme.

Shall I close this issue?

Thank you.
Denzel
